tests/integration/pilot/upgrade/goldens/golden-minimal-1.7.4.yaml

```diff
@@ -0,0 +1,8544 @@
+# GENERATED BY RUNNING "istioctl manifest generate --set profile=minimal --set revision=1-7-4" against version 1.7-4 istioctl
```

Comment:
I think we only want the pilot component or we are going to revert all the changes to the base like the CRDs and roles

Reply:
Done